assets/js/ImportModuleFromGitHub.test.js

```diff
@@ -55,6 +55,6 @@ describe('ImportModuleFromGitHub', () => {
     }, null)
     const w = wrapper(store, { api })
     expect(w.find('input')).toHaveLength(0)
-    expect(w.find('a[href^="http"]')).toHaveLength(1) // external link => to docs
+    expect(w.text()).toMatch(/workflow.workbenchbuildmodule/) // external link => to docs
```

Comment:
Because it's inside Trans, the output looks like this one below, as a result we changed it to "expect(w.find('Trans[defaults*="<0>here</0>"]')).toHaveLength(1) ", in order to have the desired result.

```
                              <div className="modal-body">
                                <div className="label-margin t-m-gray info-1">
                                  <WithI18n id="workflow.workbenchbuildmodule" defaults="Learn about how to build your own module <0>here</0>" components={{...}}>
                                    <I18n update={true} withHash={true}>
                                      <Trans id="workflow.workbenchbuildmodule" defaults="Learn about how to build your own module <0>here</0>" components={{...}} i18n={{...}} i18nHash={[undefined]}>
                                        <Render render={[undefined]} className={[undefined]} value="workflow.workbenchbuildmodule">
                                          workflow.workbenchbuildmodule
                                        </Render>
                                      </Trans>
                                    </I18n>
                                  </WithI18n>
                                </div>
                              </div>
```